usr/lib/python3/dist-packages/linuxmusterLinuxclient7/printers.py

```diff
@@ -73,16 +73,20 @@ def translateSambaToIpp(networkPath):
 # --------------------
 
 def _installPrinter(username, networkPath, name):
-    logging.debug("Installing Printer {0} on {1}".format(name, networkPath))
-    installCommand = ["lpadmin", "-p", name, "-E", "-v", networkPath, "-m", "everywhere", "-u", f"allow:{username}"]
+    logging.info("_installPrinter {0} on {1}".format(name, networkPath))
+    installCommand = ["timeout", "3", "lpadmin", "-p", name, "-E", "-v", networkPath, "-m", "everywhere", "-u", f"allow:{username}"]
     logging.debug("* running '{}'".format(" ".join(installCommand)))
-
-    if not subprocess.call(installCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE) == 0:
-        logging.fatal(f"* Error installing printer {name} on {networkPath}!\n")
-        return False
-
-    logging.debug("* Success!")
-    return True
+    p = subprocess.Popen(installCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Is it necessary to use popen here? Is there a case of some important output?

Reply:
output: stderr for debugging

Reply:
Yes, but what kind of useful information does lpadmin output? I think it is unnecessarily complicated to use popen here.

Reply:
The output is recorded in the syslog anyway. We don't need to capture it over here.

Reply:
Für die Funktionalität ist es gleichwertig, es erleicherte mir das Debuggen und ist mehrfach getestet. Auch für zukünftiges Debuggen würde ich es beibehalten.

Reply:
We use popen nowhere else. The reason being (as I already said), that all the logs are present in the syslog anyway and can be seen in linuxmuster-linuxclient7 print-logs. If we change it here, we would also have to change it in all of these places:
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/19e41e02aae7409baec47a1a5f8a9caaae414ea8/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/shares.py#L256
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/19e41e02aae7409baec47a1a5f8a9caaae414ea8/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/shares.py#L243
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/19e41e02aae7409baec47a1a5f8a9caaae414ea8/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/shares.py#L203
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/19e41e02aae7409baec47a1a5f8a9caaae414ea8/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/shares.py#L177
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/3eb7d17ec0232d51ac194a30e45399f0bf908379/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/realm.py#L17
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/3eb7d17ec0232d51ac194a30e45399f0bf908379/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/realm.py#L35
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/2ece10c6957002090e227540e1badda94a9c5808/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/imageHelper.py#L76
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/2ece10c6957002090e227540e1badda94a9c5808/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/imageHelper.py#L80
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/2ece10c6957002090e227540e1badda94a9c5808/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/imageHelper.py#L84
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/2ece10c6957002090e227540e1badda94a9c5808/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/imageHelper.py#L88
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/2ece10c6957002090e227540e1badda94a9c5808/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/imageHelper.py#L102
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/2ece10c6957002090e227540e1badda94a9c5808/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/imageHelper.py#L103
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/2ece10c6957002090e227540e1badda94a9c5808/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/imageHelper.py#L138
- https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/2ece10c6957002090e227540e1badda94a9c5808/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/imageHelper.py#L209

And probably even more. This could be done but would require separate PR. So please, use `subprocess.call` just as before.